### PR TITLE
BaSP-TG-79: Task components

### PR DESCRIPTION
**Pull request by Juanoauil**

### CODE REVIEW

File: src/Components/Projects/ListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const ListItem = ({ listItem }) => {
+  const test = listItem.map((item) => {
+    console.log('item', item);
```
Comment (cocolobos) [inline src/Components/Projects/ListItem.jsx:5]: remove console log

File: src/Components/Projects/ListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const ListItem = ({ listItem }) => {
+  const test = listItem.map((item) => {
```
Comment (cocolobos) [inline src/Components/Projects/ListItem.jsx:4]: the name should be more descriptive

File: src/Components/Projects/index.js [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+const Projects = () => {
+  const [list, setList] = useState([]);
+
+  useEffect(async () => {
+    try {
+      const response = await fetch(`https://coco-trackgenix-server.vercel.app/projects`);
+      const data = await response.json();
+      console.log(data.data);
+      setList(data.data);
+      console.log(setList);
```
Comment (cocolobos) [inline src/Components/Projects/index.js:14]: remove console log

File: src/Components/Projects/index.js [hunk 4]
```diff
@@ -1,11 +1,30 @@
...
+    } catch (error) {
+      console.log(error);
+    }
+  }, []);
+  console.log('Probando', list);
```
Comment (cocolobos) [inline src/Components/Projects/index.js:19]: remove console log

File: src/Components/Tasks/index.js [hunk 5]
```diff
@@ -1,11 +1,58 @@
...
+      <div>
+        <button onClick={showList}>{show === false ? 'Add' : 'Back'}</button>
+      </div>
+      {show === false ? (
+        <List
+          key={list.id}
+          list={list}
+          setList={setList}
+          deleteItem={deleteItem}
+          // updateItem={updateItem}
```
Comment (cocolobos) [inline src/Components/Tasks/index.js:48]: remove commented code

File: src/Components/Tasks/FormList/FormList.jsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          />
+        </div>
+        <div>
+          <button type="submit">Create</button>
+        </div>
+      </form>
+    </div>
+  );
+};
+
```
Comment (cocolobos) [inline src/Components/Tasks/FormList/FormList.jsx:68]: after adding should redirect to the list

File: src/Components/Tasks/ListItem/ListItem.jsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+const ContentList = ({ listItem, deleteItem }) => {
+  if (listItem.length === 0 || listItem === 'undefined') {
+    return 'There are not tasks';
+  }
+
+  const [updItem, setUpdItem] = useState([]);
+
+  const [openModal, setModal] = useState(false);
+
+  const updateItem = (id) => {
```
Comment (cocolobos) [inline src/Components/Tasks/ListItem/ListItem.jsx:13]: not working the editing function, also when finish editing should redirect to list component

File: src/Components/Projects/ListItem.jsx (hunk 2 above)
Comment (Juanoauil) [reply src/Components/Projects/ListItem.jsx:4]: projects are not mine, but I do it

**Review by dayanagm96 — APPROVED** [review]

**Review by cocolobos — APPROVED** [review]

**Pull request merged**